OSDOCS-13314: 4.17.16 RN (#88380)
openshift/openshift-docs: release_notes/ocp-4-17-release-notes.adoc

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.17.16 --pullspecs
----

[id="ocp-4-17-16-bug-fixes_{context}"]
==== Bug fixes

* Previously, the Bare Metal Operator (BMO) created the `HostFirmwareComponents` custom resource for all Bare Metal hosts (BMH), including ones based on the intelligent platform management interface (IPMI), which did not support it. With this release, `HostFirmwareComponents` custom resources are only created for BMH that support it. (link:https://issues.redhat.com/browse/OCPBUGS-49701[*OCPBUGS-49701*])

* Previously, importing manifest lists could cause an API crash if the source registry returned an invalid sub-manifest result. With this update, the API flags an error on the imported tag instead of crashing. (link:https://issues.redhat.com/browse/OCPBUGS-49399[*OCPBUGS-49399*])

* Previously, the Konnectivity proxy used by the `openshift-apiserver` in the control plane resolved registry names with cloud API suffixes on the control plane and then attempted to access them through the data plane. 
+

> A hosted cluster that used the no-egress feature in ROSA, as well as a container registry that was accessible through an Amazon Virtual Private Cloud (VPC) endpoint was created but failed to install because `imagestreams` that use the container registry could not resolve.  With this release, the Konnectivity proxy resolves and routes hostnames consistently. (link:https://issues.redhat.com/browse/OCPBUGS-46465[*OCPBUGS-46465*]) 

Reviewer: ```suggestion
A hosted cluster that used the no-egress feature in ROSA, and a container registry that was accessible through an Amazon Virtual Private Cloud (VPC) endpoint was created but failed to install because `imagestreams` that use the container registry did not resolve. With this release, the Konnectivity proxy resolves and routes hostnames consistently. (link:https://issues.redhat.com/browse/OCPBUGS-46465[*OCPBUGS-46465*]) 
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

// 4.17.16
[id="ocp-4-17-16_{context}"]

> === RHSA-2025:1120 - {product-title} {product-version}.16 bug fix, and security update advisory

Reviewer 2: Nit for next time:
```suggestion
=== RHSA-2025:1120 - {product-title} {product-version}.16 bug fix and security update advisory
```
